dvc/exceptions.py

```diff
@@ -227,3 +227,12 @@ def __init__(self):
         super(RecursiveAddingWhileUsingFilename, self).__init__(
             "using fname with recursive is not allowed."
         )
+
+
+class OverlappingOutputPathsError(DvcException):
+    def __init__(self, out_path_1, out_path_2):
+        super(OverlappingOutputPathsError, self).__init__(
+            "Paths for outs: \n {} \n {} \n overlap. To avoid unpredictable "
```

Comment:
It would be great if we could also specify stage files that these outputs belong to. Also, spaces before \n are not needed.